### ISSUE TITLE
Training front-end on Kitti dataset

### ISSUE THREAD

**Issue by atanas1054** [fyu/dilation#25]
Hello,

I am trying to train the front-end module on the Kitti dataset: http://adas.cvc.uab.es/s2uad/

I am using the following settings: 628x628 random cropping, reflection padding, 4 mini-batch size (can't fit 8 into memory), 0.001 learning rate, 0.99 momentum, SGD.  

The problem is that my test loss stays around 9 without changing and the train loss just jumps between 30 and 50. The test accuracy is 0. I tried increasing the iter_size without any success. Any idea what I might be doing wrong?

**Comment by huyphongdo** [comment]
@atanas1054  Hi,
Have you trained success with Kitti yet?
I can not train with Kitti data set to can you give me some guideline?
